spec/fixtures/files/prosecution_case_search_result_with_only_blank_defendant.json

```diff
@@ -0,0 +1,135 @@
+{
+  "totalResults": 1,
+  "cases": [
+    {
+      "caseStatus": "CLOSED",
+      "prosecutionCaseId": "5edd67eb-9d8c-44f2-a57e-c8d026defaa4",
+      "prosecutionCaseReference": "20GD0217100",
+      "defendantSummary": [
+        {
+          "proceedingsConcluded": true,
+          "representationOrder": {
+            "applicationReference": "LAA-20191601",
+            "effectiveFromDate": "2019-09-12",
+            "effectiveToDate": "2019-12-12",
+            "laaContractNumber": "99998888"
+          },
+          "defendantMiddleName": "Andrew",
```

Comment:
Are we also publicly leaking real prod data here? 😬

Reply:
Hopefully not... I copied it from an existing file in the same directory - It has a clearly fake defendantASN (ARREST123), but other parts may not be. I will raise this with the team next week as it's an interesting issue.

Reply:
If there was any issue with this data, then we probably need to do some sort of audit as there is at least one existing file with the same data.